content/guides/tooling/sdks/dotnet-gen.md

```diff
@@ -15,13 +15,13 @@ alias_paths: []
 
   Don’t worry, your existing code will continue to work without changes. You can still use your applications based on Box Next Generation SDKs with no impact, but you won't receive new features, updates, or bug fixes.
 
-  All future development, including new features and updates for the Next Generation SDKs, will be delivered through Box core SDKs starting with version `v10`. Currently, `v10` is available as a separate [branch][branch].
-
-  For more details, see our [SDK versioning strategy document][versioning].
+  All future development, including new features and updates for the Box Next Generation SDKs, will be delivered through Box core SDKs. The standalone generated artifact was introduced in the version `v10` of the Box core SDKs, and it is currently available as a separate [branch][sdk-branch].
 </Message>
 
 Check the [migration guide][migration] to learn how to switch from Box Next Generation SDK to Box core SDK `v10`.
 
+For more details, see our [SDK versioning strategy document][versioning].
+
 [versioning]: g://tooling/sdks/sdk-versioning
-[branch]: https://github.com/box/box-windows-sdk-v2/tree/sdk-gen
+[sdk-branch]: https://github.com/box/box-windows-sdk-v2/tree/sdk-gen
 [migration]: https://github.com/box/box-windows-sdk-v2/blob/sdk-gen/migration-guides/from-dotnet-sdk-gen-v1-to-box-windows-sdk-v10.md
```

Comment:
Link to dotnet is incorrect.

It should be

https://github.com/box/box-windows-sdk-v2/blob/sdk-gen/migration-guides/from-dotnet-sdk-gen-v1-to-box-windows-sdk.md

(no v10 at the end)